crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -261,6 +262,7 @@ async fn verify_rrsets<H>(
     handle: DnssecDnsHandle<H>,
     records: Vec<Record>,
     options: DnsRequestOptions,
+    current_time: u32,
```

Comment:
refactor idea for later: create a newtype that wraps `u32` and limits the operations that can be done on it. probably only want to allow `PartialOrd` ops and not be able to mutate it, e.g. with `Add` and `Mul` operations?

maybe that'll be a good place to implement the 'Serial Number' logic. name the newtype `SerialNumber`

Reply:
Good idea.